tests/unit/anchore_engine/services/policy_engine/engine/vulns/test_mappers.py

```diff
@@ -152,427 +152,508 @@ def test_image_content_to_grype_metadata(self, input_metadata, expected_output):
-                    ],
-                    "name": "openssl-libs",
-                    "type": "rpm",
-                    "locations": [{"path": "pkgdb"}],
-                    "version": "1.1.1g-15.el8_3",
-                    "metadataType": "RpmdbMetadata",
-                    "metadata": {
-                        "sourceRpm": None,
-                        "epoch": None,
-                    },
-                },
-                id="rpm-no-src",
-            ),
-            pytest.param(
-                ENGINE_PACKAGE_MAPPERS.get("rpm"),
-                {
-                    "cpes": [
-                        "cpe:2.3:a:openssl-libs:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl_libs:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl-libs:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl_libs:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:redhat:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:redhat:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                    ],
-                    "license": "OpenSSL and ASL 2.0",
-                    "licenses": ["OpenSSL", "and", "ASL", "2.0"],
-                    "origin": "Red Hat, Inc.",
-                    "package": "openssl-libs",
-                    "size": "3757208",
-                    "sourcepkg": None,
-                    "type": "rpm",
-                    "version": "1:1.1.1g-15.el8_3",
-                },
-                {
-                    "cpes": [
-                        "cpe:2.3:a:openssl-libs:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl_libs:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl-libs:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl_libs:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:openssl:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:redhat:openssl-libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                        "cpe:2.3:a:redhat:openssl_libs:1.1.1g-15.el8_3:*:*:*:*:*:*:*",
-                    ],
-                    "name": "openssl-libs",
-                    "type": "rpm",
-                    "locations": [{"path": "pkgdb"}],
-                    "version": "1:1.1.1g-15.el8_3",
-                    "metadataType": "RpmdbMetadata",
-                    "metadata": {
-                        "sourceRpm": None,
-                        "epoch": 1,
-                    },
+    pass
```

Comment:
Any reason why this was added and the tests are out of the class?

Reply:
hmm, this doesn't look right. I'll fix this